### PR TITLE
show display name for institute

### LINKED ISSUE #1636: UI bug
There is a tiny bug where institute name changes after selecting tab for clinvar submissions.

Steps:
1. click on institute:
image
2. click on an institute, here i clicked on "clinical genomics"
image
3. Now that clinvar submssions shows up, click on "clinvar submission". clinical genomics changes to cust000 cases:
image

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR resolves #1636 

**How to test**:
1. install on stage
1. follow instructions in #1636 

**Expected outcome**:
- [x] Institute display name displayed
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by CR

This is a patch

### CODE REVIEW

**Comment by northwestwitch** [comment]
Hi Patrik, I can review, but could you please undo the blacking of the code? We haven't blacked the whole repo yet and it is really hard to follow what you actually changed here.

We are about to release a new scout version in a few days (hopefully Thursday) and then we can black the entire code, and start fresh with the pre-commit thingy. 
Thanks!

**Comment by moonso** [comment]
Hehe yes it is really hard. Either deblack it or if there are only small changes in views you might point to these.

**Comment by patrikgrenfeldt** [comment]
The template should be reviewable as it is since black won't format it. The views.py has only two differences as visible in a diff with a blacked master: 
image
Do you still want the change unblacked or is this comparison with a blacked master of equal worth?

**Comment by northwestwitch** [comment]
> The template should be reviewable as it is since black won't format it. The views.py has only two differences as visible in a diff with a blacked master:
> image
> Do you still want the change unblacked or is this comparison with a blacked master of equal worth?

Whichever is easier for you, or you can just point me to the changes, it's not such a big PR and it's fine!

**Comment by patrikgrenfeldt** [comment]
> Whichever is easier for you, or you can just point me to the changes, it's not such a big PR and it's fine!

The changes visible in the screenshot are the only changes in view.py

**Comment by northwestwitch** [comment]
Ok, thanks, I'll review this afternoon!

**Comment by northwestwitch** [comment]
Reviewing this now!

**Comment by northwestwitch** [comment]
Works perfectly!

This is also testable locally. For clarity sake I've renamed the institute display_name to `The Umbrella Corp.`
This is before the change:
image
image

This is after:
image
image
